modules/nodes-cluster-worker-latency-profiles-using.adoc

```diff
@@ -138,3 +138,64 @@ $ oc get KubeControllerManager -o yaml | grep -i workerlatency -A 5 -B 5
 
 To change the medium profile to default or change the default to medium, edit the `node.config` object and set the `spec.workerLatencyProfile` parameter to the appropriate value.
 
+== Using worker latency profiles at cluster creation
+
+To implement a worker latency profile in the manifest, follow these steps:
+
+* Create the manifest needed to build the cluster.
```

Comment:
Think you need to do specify how to create the base manifest then you are able to edit with first command below and then should specify create cluster creation as normal 

`openshift-install create manifests`

`openshift-install create `

Reply:
@paigerube14 I can't repeat that here because we have an installation section that goes into that, with subsections targeted toward every supported kind of installation:  https://docs.openshift.com/container-platform/4.13/installing/index.html

Reply:
we need the `openshift-install create manifests` to create the default node manifest yaml (and other files) before we add in the workerLatencyProfile. It needs a specific set up of folders to be able to do the installation properly